furusiya/model/components/fighter.py

```diff
@@ -62,4 +52,23 @@ def heal(self, amount):
         # heal by the given amount, without going over the maximum
         self.hp += amount
         if self.hp > self.max_hp:
-            self.hp = self.max_hp
\ No newline at end of file
+            self.hp = self.max_hp
+
+    def die(self):
```

Comment:
I see two missing tests: dying drops arrows, and dying calls death function if defined.